binding/form_mapping.go

```diff
@@ -148,16 +150,22 @@ func setFloatField(val string, bitSize int, field reflect.Value) error {
 }
 
 func setTimeField(val string, structField reflect.StructField, value reflect.Value) error {
-	timeFormat := structField.Tag.Get("time_format")
-	if timeFormat == "" {
-		return errors.New("Blank time format")
-	}
 
 	if val == "" {
 		value.Set(reflect.ValueOf(time.Time{}))
 		return nil
 	}
 
+	timeFormat := structField.Tag.Get("time_format")
+	if timeFormat == "" {
+		t, err := dateparse.ParseAny(val)
```

Comment:
About the separate lib the only benefit is that it uses a byte reader with state switch to determine the format to be more efficient.